### PR TITLE
Fix for 2346. Guide people towards the Name= syntax for connection strings

### LINKED ISSUE #2346: Guide People to Use the Name= Syntax for Connection Strings
See [this comment](https://github.com/dotnet/efcore/issues/19899#issuecomment-586426274).

We should guide people better towards using the `Name=<database-alias>` syntax for connection strings.

### PR DESCRIPTION
**Pull request by lajones**
Fixes #2346 .

Guide people towards the Name= syntax for connection strings.

### CODE REVIEW

File: entity-framework/core/miscellaneous/connection-strings.md [hunk 1]
```diff
@@ -61,7 +61,16 @@ public class BloggingContext : DbContext
 
 ## ASP.NET Core
 
-In ASP.NET Core the configuration system is very flexible, and the connection string could be stored in `appsettings.json`, an environment variable, the user secret store, or another configuration source. See the [Configuration section of the ASP.NET Core documentation](https://docs.asp.net/en/latest/fundamentals/configuration.html) for more details. The following example shows the connection string stored in `appsettings.json`.
+In ASP.NET Core the configuration system is very flexible, and the connection string could be stored in `appsettings.json`, an environment variable, the user secret store, or another configuration source. See the [Configuration section of the ASP.NET Core documentation](https://docs.asp.net/en/latest/fundamentals/configuration.html) for more details.
```
Comment (smitpatel) [inline entity-framework/core/miscellaneous/connection-strings.md:64]: please convert above link to docs.microsoft.com. It already redirects.

File: entity-framework/core/miscellaneous/connection-strings.md [hunk 2]
```diff
@@ -61,7 +61,16 @@ public class BloggingContext : DbContext
...
+
+For instance, you can use the [Secret Manager tool](https://docs.microsoft.com/aspnet/core/security/app-secrets#secret-manager) to store your database password and then, in scaffolding, use a connection string that simply consists of `Name=<database-alias>`.
```
Comment (smitpatel) [inline entity-framework/core/miscellaneous/connection-strings.md:66]: Follow our guidelines for links. Base domain does not need to be specified for docs.microsoft.com.

Comment (lajones) [reply entity-framework/core/miscellaneous/connection-strings.md:66]: @smitpatel I'm not seeing this spelled out in the guidelines - can you point me to what rules you are following? And an example would be helpful. Note this is not in the entity-framework docs.

Comment (smitpatel) [reply entity-framework/core/miscellaneous/connection-strings.md:66]: https://github.com/dotnet/EntityFramework.Docs/issues/1900

Comment (lajones) [reply entity-framework/core/miscellaneous/connection-strings.md:66]: @smitpatel Created PR #2351 for this.

**Pull request merged**